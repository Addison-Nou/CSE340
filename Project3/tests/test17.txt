{
    x1, x2, x3, x4 , x5 : INT;
         
    x3 = 2;

    WHILE ( < x1 100 ) {
        x1 = + x1 1;
        x2 = 1;
        x4 = + x2 1;
        x1 = + x3 1;
        WHILE ( < x1 10 ) {
            x5 = + x1 + x3 x5 ;
        }
        x1 = x5;
        x5 = 1;
    }
    
    x2 = + x2 1 ;
}