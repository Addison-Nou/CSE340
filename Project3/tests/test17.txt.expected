UNINITIALIZED x1 6
UNINITIALIZED x1 7
UNINITIALIZED x5 12
UNINITIALIZED x5 14
UNINITIALIZED x2 18